internal/applicationsnapshot/report.go

```diff
@@ -235,6 +239,51 @@ func condensedMsg(results []evaluator.Result) map[string][]string {
 	return shortNames
 }
 
+func generateMarkdownSummary(r *Report) ([]byte, error) {
+	var markdownBuffer bytes.Buffer
+	markdownBuffer.WriteString("| Field     | Value |Status|\n")
+	markdownBuffer.WriteString("|-----------|-------|-------|\n")
+
+	var totalViolations, totalWarnings, totalSuccesses int
+	pr := r.toSummary()
+	for _, component := range pr.Components {
+		totalViolations += component.TotalViolations
+		totalWarnings += component.TotalWarnings
+		totalSuccesses += component.TotalSuccesses
+	}
+	type Field struct {
+		Name  string
+		Value string
+	}
+	fields := []Field{
+		{"Time", r.created.UTC().Format("2006-01-02 15:04:05")},
+		{"Name", " "},
+		{"Successes", fmt.Sprint(totalSuccesses)},
+		{"Failures", fmt.Sprint(totalViolations)},
+		{"Warnings", fmt.Sprint(totalWarnings)},
+		{"Result", fmt.Sprint(pr.Success)},
+	}
+
+	for _, field := range fields {
+		writeMarkdownField(&markdownBuffer, field.Name, field.Value)
+	}
```

Comment:
Perhaps the `Field` struct is not needed...
```suggestion
	writeMarkdownField(&markdownBuffer, "Time", r.created.UTC().Format("2006-01-02 15:04:05")
	writeMarkdownField(&markdownBuffer, "Name", " ")
	...
```